src/test/java/com/sonyericsson/jenkins/plugins/bfa/db/EmbeddedMongoStatisticsTest.java

```diff
@@ -156,9 +157,11 @@ private void setUpTwoCauses() throws Exception {
      * @throws Exception if something goes wrong
      */
     @Test
+    @Ignore("Fails on new version of the mongo java driver - returns null instead of expected date")
```

Comment:
not sure what to do here, spent awhile on it and the existing code doesn't seem to work on the new driver.

Reply:
fixed